apps/client/src/components/Conclusion/NeedPermit.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Heading, Paragraph } from "@datapunt/asc-ui";
+import React from "react";
+
+import ComponentWrapper from "../../atoms/ComponentWrapper";
+import { Olo } from "../../config";
+import { actions, eventNames } from "../../config/matomo";
```

Comment:
I've did a lot of refactors in my upcoming PR in this file. Can you please un-rename this? 😛